pkg/cmd/roachtest/tests/util.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/util/retry"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/cockroachdb/errors"
+)
+
+// WaitFor3XReplication waits until all ranges in the system are on at least
+// three voters.
+//
+// TODO(nvanbenschoten): this function should take a context and be responsive
+// to context cancellation.
+func WaitFor3XReplication(t test.Test, db *sql.DB) {
```

Comment:
Like the name change. Should eventually have a variant that detects the expected number of replicas.